pipeline/scripts/13-excelExport.R

```diff
@@ -137,9 +144,44 @@ if (z_score == 1) {
   }
   colnames(outlist)[startcol:ncol(outlist)] <- cnames.z
   
+  # Extra output: calculate robust scaler (Zscores minus outliers in Controls)
+  # calculate mean and sd for Control group without outliers
+  outlist.noZ$avg.ctrls <- 0
+  outlist.noZ$sd.ctrls  <- 0
+  
+  robustscaler <- function(x, perc=5, control_col_ids) {
```

Comment:
Wat is x?

Reply:
x zijn de intensiteiten van de controles, ik heb er nu control_intensities van gemaakt.